src/Data/Array.purs

```diff
@@ -600,6 +601,16 @@ foreign import partition
   -> Array a
   -> { yes :: Array a, no :: Array a }
 
+-- | Splits an array into two pieces, where the first array has `n` elements
+-- | and the second array has the remaining elements.
+-- |
+-- | ```purescript
+-- | splitAt 3 [1, 2, 3, 4, 5] == Tuple [1, 2, 3] [4, 5]
+-- | ```
```

Comment:
Docs should specify what happens when an empty array is the argument

Reply:
Added in 6f414d6.